### ISSUE TITLE
lug-helper.sh: 181: Syntax error: "(" unexpected

### ISSUE THREAD

**Issue by JustinHowes01** [starcitizen-lug/lug-helper#75]
**Describe the bug**
When trying to run LUG, nothing happens. I ran it in the terminal, and this is the error it produced

**To Reproduce**
1. Extract LUG after installation
2. Run lug-helper.sh

**Expected behavior**
LUG GUI launches

**OS Info:**
 - Distro: Kubuntu
 - Version: 23.10

**Comment by Termuellinator** [comment]
That's odd - possibly a shell specific behavior? How exactly are you running it? And which shell are you using (i think (K)ubuntu defaults to bash?)?

**Comment by JustinHowes01** [comment]
> That's odd - possibly a shell specific behavior? How exactly are you running it? And which shell are you using (i think (K)ubuntu defaults to bash?)?

I ran it using "sh lug-helper.sh", and it gave me the error. I actually ran it with bash shortly after I posted the issue, and I don't get the error with it.

However, I did read in the ReadMe that zenity is a dependency for the gui, and I've figured out that it wasn't installed. 

It was completely my mistake for overlooking that piece of info, and everything is working 100% now. Should I delete this issue, if possible?
